packages/react-dom/src/client/ReactDOMComponent.js

```diff
@@ -1348,6 +1348,10 @@ export function listenToEventResponderEventTypes(
       const targetEventType = isPassive
         ? eventType
         : eventType.substring(0, eventType.length - 7);
+      // We don't listen to this as we actually emulate it in the host config
+      if (targetEventType === 'beforeblur') {
```

Comment:
Seems kind of unfortunate to have a specific event type hard-coded in this loop like this. Do you expect we may be adding more checks like this one over time?

Reply:
We will be killing Flare soon, so this should all go away. Mwhuhaha.